docs/getting-started-guides/minikube.md

```diff
@@ -0,0 +1,76 @@
+---
+---
+
+* TOC
+{:toc}
+
+### Requirements
+
+Minikube requires that VT-x/AMD-v virtualization is enabled in BIOS.
+
+#### Linux
+
+Minikube requires the latest [Virtualbox](https://www.virtualbox.org/wiki/Downloads) to be installed on your system.
+
+#### OSX
+
+Minikube requires one of the following:
+
+* The latest [Virtualbox](https://www.virtualbox.org/wiki/Downloads).
+* The latest version of [VMWare Fusion](https://www.vmware.com/products/fusion).
+
+### Installation
+
+See the [latest Minikube release](https://github.com/kubernetes/minikube/releases) for installation instructions.
+
+### Starting the cluster
+
+To start a cluster, run the command:
+
+```shell
+minikube start
+Starting local Kubernetes cluster...
+Kubernetes is available at https://192.168.99.100:443.
+```
+
+This will build and start a lightweight local cluster, consisting of a master, etcd, Docker and a single node.
+
+Minikube will also create a "minikube" context, and set it to default in kubectl.
+To switch back to this context later, run this command: `kubectl config use-context minikube`.
+
+Type `minikube stop` to shut the cluster down.
+
+Minikube also includes the [Kubernetes dashboard](http://kubernetes.io/docs/user-guide/ui/). Run this command to see the included kube-system pods:
+
+```shell
+$ kubectl get pods --all-namespaces
+NAMESPACE     NAME                           READY     STATUS    RESTARTS   AGE
+kube-system   kube-addon-manager-127.0.0.1   1/1       Running   0          35s
+kube-system   kubernetes-dashboard-9brhv     1/1       Running   0          20s
+```
+
+Run this command to open the Kubernetes dashboard:
+
+```shell
+minikube dashboard
+```
+
+### Test it out
+
+List the nodes in your cluster by running:
+
+```shell
+kubectl get nodes
+```
+
+Use the built-in Docker daemon with:
```

Comment:
Not clear what this means.  Would be helpful to give context and when you would need this.